packages/core-utils/src/itinerary.js

```diff
@@ -480,36 +484,45 @@ export function getTransitFare(fareComponent) {
   const dollarsToString = dollars => `${symbol}${dollars.toFixed(2)}`;
   return {
     centsToString,
+    currencyCode,
     dollarsToString,
     transitFare
   };
 }
 
 /**
  * For an itinerary, calculates the transit/TNC fares and returns an object with
- * these values as well as string formatters.
+ * these values, currency info, as well as string formatters.
  */
 export function calculateFares(itinerary) {
   // Extract fare total from itinerary fares.
   const fareComponent =
     itinerary.fare && itinerary.fare.fare && itinerary.fare.fare.regular;
   // Get string formatters and itinerary fare.
-  const { centsToString, dollarsToString, transitFare } = getTransitFare(
-    fareComponent
-  );
+  const {
+    centsToString,
+    currencyCode: transitCurrencyCode,
+    dollarsToString,
+    transitFare
+  } = getTransitFare(fareComponent);
   // Process any TNC fares
   let minTNCFare = 0;
   let maxTNCFare = 0;
+  let tncCurrencyCode;
   itinerary.legs.forEach(leg => {
     if (leg.mode === "CAR" && leg.hailedCar && leg.tncData) {
-      const { maxCost, minCost } = leg.tncData;
+      const { currency, maxCost, minCost } = leg.tncData;
       // TODO: Support non-USD
       minTNCFare += minCost;
       maxTNCFare += maxCost;
+      tncCurrencyCode = currency;
     }
   });
+
   return {
     centsToString,
+    // Hopefully, the currency code is consistent throughout.
```

Comment:
I think it might be better to explicitly state in the JSDoc of the `calculateFares` funciton that fares from an itinerary are all assumed to be in the same currency.

Reply:
Yep, thanks, changed in https://github.com/opentripplanner/otp-ui/pull/282/commits/72d732d80a97f84b160024ab08460adcff6fcb1b.